python/ray/train/base_trainer.py

```diff
@@ -362,6 +366,23 @@ def fit(self) -> Result:
             raise TrainingFailedError from e
         return result
 
+    def _extract_fields_for_tuner_param_space(self) -> Dict:
+        """Extracts fields to be included in `Tuner.param_space`.
+
+        This is needed to leverage the full logging/integration offerings from Tune.
+        For example, `param_space` is logged automatically to wandb integration.
+
+        Currently only done for `train_loop_config`.
+
+        Returns:
+            A dictionary that should be passed to Tuner.param_space.
+        """
+        result = {}
+        for key in ["train_loop_config"]:
```

Comment:
Should this be a `Trainer` class property so we can extend this in other trainers?